### DISCUSSION TITLE
Rest client deletes body content

### DISCUSSION THREAD

**Discussion by francescobande** [resteasy/resteasy#4153] [Q&A]
We are using resteasy-client to consume our API.
We observed this behaviour: if our API returns an error status with a body, the latter is present when I call directly the API (e.g. by using curl or swagger-ui), but it is not present when I consume it using your client.
By debugging your library, I noticed that the below mentioned code does not take the response's entity into consideration. 

https://github.com/resteasy/resteasy/blob/d8d92458f7a5af2f4d0bd22d4e78c86327e6f10a/resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/engines/ManualClosingApacheHttpClient43Engine.java#L448-L453

I was wondering if this had been done on purpose or, if that is not the case, it might be added.

**Comment by jamezp** [comment]
What version of RESTEasy are you using?

The body/content is set further up in https://github.com/resteasy/resteasy/blob/d8d92458f7a5af2f4d0bd22d4e78c86327e6f10a/resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/engines/ManualClosingApacheHttpClient43Engine.java#L375-L394.

You could potentially be seeing the result of https://issues.redhat.com/browse/RESTEASY-2825. However, `curl` should show the same thing.
